controllers/articlesReactionController.js

```diff
@@ -0,0 +1,98 @@
+const {
+  getUserReaction,
+  createReaction,
+  updateReaction,
+  removeReaction,
+  getTotalReactions
+} = require('../lib/modelManagers/articlesReactionModel');
+
+const {
+  SERVER_ERROR_MESSAGE,
+  RESET_REACTION,
+  RESOURCE_CREATED_CODE,
+  ARTICLE_REACTION_STATUS,
+  OK_CODE
+} = require('../constants');
+const {
+  successResponse,
+  serverFailure,
+  formatReaction,
+  formatMessage
+} = require('../lib/utils/messageHandler');
+
+let newReaction;
+/**
+ *
+ *
+ * @class commentsReactionController
+ */
+class ArticlesReactionController {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @memberof ArticlesReactionController
+   * @returns {object} response of the request
+   */
+  static async allReactions(req, res) {
+    const message = formatMessage(req.params.reaction);
+    try {
+      const reactions = await getTotalReactions(req.params.articleId, req.params.reaction);
+      const { rows, ...count } = reactions;
```

Comment:
The rest operation here is not needed
```suggestion
      const { rows, count } = reactions;
```

Reply:
This is now gone because of the change I made above